src/components/fullscreen/_fullscreen.scss

```diff
@@ -1,18 +1,26 @@
-.control-button-container.control-fullscreen {
-  .control-button {
-    transition: 100ms transform;
-    transform: scale(1);
+.in-browser-fullscreen-mode {
+  width: auto !important;
```

Comment:
Need to also apply left/top 0 and z-index to max in case other elements have higher position